### DISCUSSION TITLE
From the child element in an array I only want the first select the first child

### DISCUSSION THREAD

**Discussion by f3bw** [tailwindlabs/tailwindcss#10942] [Help] [answered]
### First of the child elements
I want to select the first, but only the first of all the child elements. What is the correct syntax for this?

My current try is `[&>*]:first:p-0`
### Example 
```
 <Content className="odd:bg-surface-50 overflow-hidden text-base text-brand-black data-[state=open]:animate-slideDown data-[state=closed]:animate-slideUp [&>*]:first:p-0">
      <div className="p-md">{children}</div>
    </Content>
```

**Comment by wongjn** [comment, marked as answer]
`[&_:first-child]:p-0`

**Reply by olets** [reply]
That will have unexpected results for nested content https://play.tailwindcss.com/6rKuMT7ecE

**Reply by wongjn** [reply]
Indeed, but it fulfils the request to <q>select the first, but only the first of **all** the child elements.</q> (emphasis mine).

**Reply by olets** [reply]
Ah yes indeed
